commands/Admin/customcmd.js

```diff
@@ -0,0 +1,86 @@
+    }
+
+    async create(msg, [name, ...content]) {
+        if (this.client.commands.has(name)) return msg.reply(`${this.client.emotes.cross} ***\`${name}\` ${msg.language.get("MESSAGE_CMD_EXISTS")}***`);
+        const cmd = msg.guild.settings.customcmds.cmds.find(c => c.name === name);
+        if (cmd) return msg.reply(`${this.client.emotes.cross} ***\`${name}\` ${msg.language.get("MESSAGE_CMD_EXISTS")}***`);
+        await msg.guild.settings.update("customcmds.cmds", { content: content.join(" "), name: name });
+        return msg.sendMessage(`${this.client.emotes.check} ***\`${name}\` ${msg.language.get("MESSAGE_CMD_ADDED")} ${msg.author.tag}!***`);
+    }
+
+    async delete(msg, [name]) {
+        const cmd = msg.guild.settings.customcmds.cmds.find(c => c.name === name);
+        if (!cmd) return msg.reply(`${this.client.emotes.cross} ***\`${name}\` ${msg.language.get("MESSAGE_CMD_NOTFOUND")}***`);
+        await msg.guild.settings.update("customcmds.cmds", cmd, { action: `remove` });
+        return msg.sendMessage(`${this.client.emotes.check} ***\`${name}\` ${msg.language.get("MESSAGE_CMD_REMOVED")} ${msg.author.tag}!***`);
+    }
+
+    async toggle(msg) {
+        if (msg.guild.settings.get("customcmds.enabled") === false) {
+            return msg.guild.settings.update("customcmds.enabled", true).then(() => {
+                msg.sendMessage(`${this.client.emotes.check} ***${msg.language.get("MESSAGE_COMMAND_CUSTOM_ENABLED")}***`);
+            });
+        } else {
+            return msg.guild.settings.update("customcmds.enabled", false).then(() => {
+                msg.sendMessage(`${this.client.emotes.cross} ***${msg.language.get("MESSAGE_COMMAND_CUSTOM_DISABLED")}***`);
+            });
+        }
+    }
+
+    async update(msg, [name, ...content]) {
+        if (this.client.commands.has(name)) return msg.reply(`${this.client.emotes.cross} ***\`${name}\` ${msg.language.get("MESSAGE_CMD_EXISTS")}***`);
+        const cmd = msg.guild.settings.customcmds.cmds.find(c => c.name === name);
+        if (cmd) {
+            await msg.guild.settings.update("customcmds.cmds", cmd, { action: "remove" }).then(() => {
+                msg.guild.settings.update("customcmds.cmds", { content: content.join(" "), name: cmd.name }, { action: "add" });
+            });
+            return msg.sendMessage(`${this.client.emotes.check} ***\`${name}\` ${msg.language.get("MESSAGE_CMD_UPDATED")} ${msg.author.tag}!***`);
+        } else {
+            return msg.reply(`${this.client.emotes.cross} ***\`${name}\` ${msg.language.get("MESSAGE_CMD_NOTFOUND")}***`);
+        }
+    }
+
+    async list(msg) {
+        if (msg.guild.settings.customcmds.cmds[0] === undefined) return msg.reply(`${this.client.emotes.cross} ***${msg.language.get("MESSAGE_NO_CMDS")}***`);
+        const prefix = msg.guild.settings.get("prefix");
+        const names = msg.guild.settings.customcmds.cmds.map(cmd => cmd.name);
+
+        const cmds = new RichDisplay(new MessageEmbed()
```

Comment:
Pengu will not always have perms required for RichDisplay (manage messages, add reactions). You should check for them and fallback to a normal message.